shell/src/main/java/tachyon/shell/command/CommandUtils.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.shell.command;
+
+import java.io.IOException;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.Date;
+import java.util.List;
+
+import tachyon.Constants;
+import tachyon.TachyonURI;
+import tachyon.client.file.TachyonFile;
+import tachyon.client.file.TachyonFileSystem;
+import tachyon.client.file.options.SetStateOptions;
+import tachyon.client.lineage.TachyonLineage;
+import tachyon.exception.TachyonException;
+import tachyon.thrift.FileInfo;
+import tachyon.thrift.LineageInfo;
+import tachyon.util.FormatUtils;
+
+/**
+ * Common util methods for executing commands.
+ */
+public class CommandUtils {
+
+  private CommandUtils() {
+    // Not intended for instantiation.
+  }
+
+  /**
+   * Set a new TTL value or unset an existing TTL value for file at path.
+   *
+   * @param path the file path
+   * @param ttlMs the TTL (time to live) value to use; it identifies duration (in milliseconds) the
+   *        created file should be kept around before it is automatically deleted, irrespective of
+   *        whether the file is pinned; {@link Constants#NO_TTL} means to unset the TTL value
+   * @throws IOException when failing to set/unset the TTL
+   */
+  public static void setTTL(TachyonFileSystem tfs, TachyonURI path, long ttlMs) throws IOException {
+    try {
+      TachyonFile fd = tfs.open(path);
+      SetStateOptions options = new SetStateOptions.Builder().setTTL(ttlMs).build();
+      tfs.setState(fd, options);
+    } catch (TachyonException e) {
+      throw new IOException(e.getMessage());
+    }
+  }
+
+  /**
+   * Displays information for all directories and files directly under the path specified in argv.
+   *
+   * @param tfs The TachyonFileSystem client
+   * @param path The TachyonURI path as the input of the command
+   * @param recursive Whether list the path recursively
+   * @throws IOException
+   */
+  public static void ls(TachyonFileSystem tfs, TachyonURI path, boolean recursive)
```

Comment:
This method is shared by ls and lsr, before it was duplicated in the two methods in TfsShell, I extracted it as a common method that can be used by both of them.